CHANGELOG.md

```diff
@@ -2,6 +2,10 @@ postgresql Cookbook CHANGELOG
 =============================
 This file is used to list changes made in each version of the postgresql cookbook.
 
+v4.0.7
```

Comment:
Can you revert this bit. We'll handle the changelog. This confuses people who see it and think that 4.0.7 is pushed.